examples/post-sync/.dockerignore

```diff
@@ -0,0 +1,5 @@
+# Ignore binary
+sync-status-watch
+controller_test.go
```

Comment:
nit: suggest ingoring all _test.go files. Not sure what the prefix is for it tho. * or maybe **. check the .dockerignore docs.

Reply:
done